.github/workflows/ci.yml

```
@@ -12,6 +12,10 @@ jobs:
   verify:
     name: Verify
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        ls -la
```

actionlint (1.7.3)

"run" section is scalar node but mapping node is expected

(syntax-check)